src/core/amp-mode.d.ts

```diff
@@ -0,0 +1,12 @@
+export {};
+
+declare global {
```

Comment:
mode-global.d.ts?

Reply:
Right now, this contains `AmpModeObject` and `__AMP_MODE`. I think this name is "correct" for now, and we should separately consider removing/renaming anything in core that is AMP-specific